esm_runscripts/batch_system.py

```diff
@@ -93,17 +93,37 @@ def get_batch_header(config):
         ]
         if config["general"]["jobtype"] in ["compute", "tidy_and_resume"]:
             conditional_flags.append("exclusive_flag")
+
         for flag in conditional_flags:
```

Comment:
I think this whole loop takes care of getting rid of the empty flags. If that is correct I will add a comment here:
`# Avoid using empty conditional flags (strings and list of strings supported)`.